hm-basic-webapp/templates/worker_deployment.yaml

```diff
@@ -22,14 +24,18 @@ spec:
         {{- .Values.imagePullSecrets | toYaml | nindent 8 }}
       containers:
         - image: "{{ .Values.image.repository }}:{{ .Values.ci.deploymentTag | default .Chart.AppVersion }}"
-          name: {{ include "hm-basic-webapp.prefix" . }}-worker
+          name: {{ include "hm-basic-webapp.prefix" . }}-{{ $worker.name }}-worker
           command:
-            {{- range $cmd := .Values.worker.command }}
+            {{- range $cmd := $worker.command }}
               - {{ $cmd }}
             {{- end }}
           env:
             - name: WORKER
```

Comment:
Could you remove this special env? It's redundant since we can pass the env now 🙌🏼

Reply:
Done